apps/public/src/app/(map)/locatie/kies/_components/SideBar/SideBar.tsx

```diff
@@ -40,10 +94,33 @@ export const SideBar = ({ coordinates }: Props) => {
         </Paragraph>
       </div>
       <div>
-        <Heading level={2} size="level-4">
-          Zoek op adres
-        </Heading>
-        {address && <div>{address}</div>}
+        <Field>
+          <HUILabel as={Label}>Zoek op adres</HUILabel>
+          <Combobox value={address} onChange={setAddress} onClose={() => fetchAddressList('')}>
+            <ComboboxInput
+              as={TextInput}
+              displayValue={(item: Address) => item?.weergave_naam}
+              name="address"
+              onChange={(event) => fetchAddressList(event.target.value)}
+              autoComplete="off"
+            />
+            {showListBox && (
+              <ComboboxOptions as={ListBox} modal={false}>
+                {addressList.length > 0 ? (
+                  addressList.map((test) => (
+                    <ComboboxOption key={test.id} value={test} as={ListBox.Option}>
+                      {test.weergave_naam}
+                    </ComboboxOption>
+                  ))
+                ) : (
+                  <ComboboxOption value="" disabled as={ListBox.Option}>
+                    Geen resultaten gevonden
+                  </ComboboxOption>
+                )}
+              </ComboboxOptions>
+            )}
+          </Combobox>
+        </Field>
```

Comment:
Personally I'd create a new component for this to keep the components (and tests) small, simple and easy to read.

Reply:
Good idea. What should we call it? `AddressComboBox` or something?

Reply:
yes sound good.

Reply:
I extracted this!